glommio/src/io/read_result.rs

```diff
@@ -137,3 +137,14 @@ impl ReadResultInner {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
I'm a fan of testing expected struct sizes, although it is a bit sad in my opinion that rust doesn't seem to have the equivalent of `static_assert`, which is how I'd do this in c++